test/node-only/__snapshots__/sendLayoutHTTPResponse-node.test.js.snap

```diff
@@ -109,12 +111,14 @@ exports[`sendLayoutHTTPResponse response body allows for promises to be returned
       <application name="header">
       </application>
       <div class="main-content">
-        <button>
-          Top level content
-        </button>
-        <application name="app1">
-        </application>
-        Some raw text
+        <route path="app1">
```

Comment:
Notice how the `<route>` element is new here - this is desired and expected since the browser needs this to know which application to mount.